public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
the html file should source a javascript script for the react to work

Reply:
Hmm.  I think that the problem you've run into is maybe that I have the package.json and node_modules in .gitignore;  should they not be?

Reply:
I've updated the .gitignore file.  Let me know if it works now.

Reply:
node_modules has to be ignored
package.json should be included, as well as package-lock.json, if you have it.
I can't currently read the diff of the PR on github because there are so many files and it crashes
can you remove node_modules and add it back to .gitignore push out another commit ( or revert/rebase to an earlier commit and just remove package.json from .gitignore and add it ) ?

Reply:
Fixed.